### PR TITLE
Recommendation flow implemented, changes done for reflection flow

### PR DESCRIPTION
**Pull request by Vinod-V3**

### CODE REVIEW

File: src/app/generic-listing-page/generic-listing-page.component.ts [hunk 1]
```diff
@@ -66,11 +68,16 @@ export class GenericListingPageComponent  implements OnInit {
...
+    if(this.pageConfig?.type == "recommendation"){
+      url = `${this.pageConfig.apiUrl}?page=${this.page}&limit=${this.limit}`
+      serviceToTrigger = this.gwApiService 
+    }
+    serviceToTrigger.get(url).subscribe({
       next: async(response: any)=>{
       await this.loaderService.dismissLoading()
       this.noData=false;
-      if(response.status == 200){
+      if(response){
```
Comment (5Amogh) [inline src/app/generic-listing-page/generic-listing-page.component.ts:80]: Hope this doesn't cause issues when the response is other than 200

File: src/app/mi-details/recommendation-details/recommendation-details.page.html
Comment (5Amogh) [inline src/app/mi-details/recommendation-details/recommendation-details.page.html:1]: Just use safety operator for few things which could cause issues

File: src/app/generic-listing-page/generic-listing-page.component.ts (hunk 1 above)
Comment (Vinod-V3) [reply src/app/generic-listing-page/generic-listing-page.component.ts:80]: This works, but inside if block I will handle it

File: src/app/mi-details/recommendation-details/recommendation-details.page.html
Comment (Vinod-V3) [reply src/app/mi-details/recommendation-details/recommendation-details.page.html:1]: Sure

**Pull request merged**